actioncable/actioncable.gemspec

```diff
@@ -10,6 +10,7 @@ Gem::Specification.new do |s|
   s.description = "Structure many real-time application concerns into channels over a single WebSocket connection."
 
   s.required_ruby_version = ">= 2.2.2"
+  s.required_rubygems_version = ">= 2.6.3"
```

Comment:
This seems pretty recent compared to ruby 2.2.2. Maybe use the new exception where available, but not insist it's there?